### ISSUE TITLE
ToDos

### ISSUE THREAD

**Issue by edwardkenfox** [edwardkenfox/welcome-to-the-genre#10]
## audio analysis

- [ ] experiment [genreXpose](https://github.com/jazdev/genreXpose) and make it working
- [ ] decide input/output I/F 

## server

- ~~[ ] setup JSON API~~
   - will be done with HTML form for now
- [x] handle audio file given with POST params body and pass it to audio analysis
- [ ] convert audio file format/encoding if necessary

## frontend

- [x] allow sending audio file with fetch and handle JSON response
   - will be done with HTML form for now
- [x] show result
- [x] enable using mic to record and send audio file
   - https://github.com/edwardkenfox/welcome-to-the-genre/pull/15
- [x] fix design
   - https://github.com/edwardkenfox/welcome-to-the-genre/issues/13

## infrastructure

- [ ] setup ec2 instance 
- [ ] make everything up and running 
- [ ] store file cache data somewhere

## nice to have

- [ ] dockerize environment
- [ ] setup build process on CI

**Comment by nwnsns** [comment]
I think I might be able to do them !

**Comment by susumunara** [comment]
・experiment genreXpose and make it working
・show result
・fix design
・setup ec2 instance
・store file cache data somewhere

・ dockerize environment

I think I might be able to them !!

**Comment by edwardkenfox** [comment]
@nwnsns Cool, then please start working on them and let me know whenever you make progress or have a problem! Also, let's have the checkboxes checked once the task is finished. 

BTW, I think it's a good idea to take full advantage of markdown expression. Take a look at the [GitHub markdown cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) when you have the time. No need to hurry.
